test/utm_params_test.rb

```diff
@@ -30,6 +30,23 @@ def test_nested
     assert_body '<img src="image.png"></a>', message
   end
 
+  # When nokogiri parses with html5, it allows an <a> tag to wrap a <table> tag
+  def test_nested_table_html5
+    with_html5 do
+      message = UtmParamsMailer.nested_table.deliver_now
+      assert_body "utm_medium=email", message
+      assert_body '<table></table></a>', message
+    end
+  end
+
+  # When nokogiri parses with html4, it disallows an <a> tag to wrap a <table> tag,
```

Comment:
I doubt anybody wants this behaviour: I'm just testing it for the sake of capturing what was already happening to show that nothing changes if you don't opt-in to html4